pkg/monitor/monitorapi/types.go

```diff
@@ -141,6 +143,8 @@ const (
 	LocatorRowKey                   LocatorKey = "row"
 	LocatorServerKey                LocatorKey = "server"
 	LocatorMetricKey                LocatorKey = "metric"
+
+	LocatorAPIUnreachableSourceKey LocatorKey = "source"
```

Comment:
A little overloaded with interval.Source. Could we come up with a better name? Is "host" ok given that's what it's sourced from on the metric. (even if it's mutated)